packages/datadog_grpc_interceptor/README.md

```diff
@@ -0,0 +1,53 @@
+
+# Datadog gRPC Interceptor / Plugin
+
+> A plugin for use with the `DatadogSdk`, used to track performance of gRPC and enable Datadog Distributed Tracing.
+
+> ⚠️ This plugin is still in Alpha / Developer Preview. 
+
+## Getting started
+
+To utilize this plugin, create an instance of `DatadogGrpcInterceptor`, then pass it to your generated gRPC client:
```

Comment:
```suggestion
To use this plugin, create an instance of `DatadogGrpcInterceptor` and pass it to your generated gRPC client:
```

Reply:
I'll make this change as a separate pull request. The grpc interceptor isn't actually being released right now.